README.adoc

```diff
@@ -66,81 +63,51 @@ You need to have Docker installed. For installation instructions, refer to the o
 
-Instead of using `thenAcceptAsync()`, `Observable` uses the [hotspot=subscribe file=0]`subscribe()` method to asynchronously process data. Thus, any required processing of data will be performed in the [hotspot=subscribe file=0]`subscribe()` method. In this case, it is simply saving the data in the temporary [hotspot=holder hotspot=holderClass file=0]`holder`. The temporary [hotspot=holder hotspot=holderClass file=0]`holder` is used to store the value returned from the client because values cannot be returned inside [hotspot=subscribe file=0]`subscribe()`. The highest and lowest load systems are updated in the [hotspot=updateHighestCall hotspot=updateHighestMethod file=0]`updateHighest()` and [hotspot=updateLowestCall hotspot=updateLowestMethod file=0]`updateLowest()` methods respectively.
-
-A [hotspot=countdownlatch file=0]`CountDownLatch` is used to keep track of how many asynchronous requests are being waited on. The [hotspot=countdownlatch file=0]`CountDownLatch` will countdown towards 0 after the completion of a thread via the [hotspot=countdown hotspot=countdown2 file=0]`countdown` method call. This means that the value will only return once the thread that's retrieving the value is complete. The [hotspot=await file=0]`await` method stops and waits until all of the requests have completed. While waiting for this countdown to complete, the main thread is free to perform other tasks. In this case, no such task is present, but a simple task can be given to occupy the thread.
-
-// =================================================================================================
-// Building the application
-// =================================================================================================
+A [hotspot=countdownlatch file=1]`CountDownLatch` object is used to keep track of how many asynchronous requests are being waited on. 
+The `CountDownLatch` object will countdown towards 0 after each completion of a thread via the [hotspot=countdown1 hotspot=countdown2 file=1]`countdown` method call. 
+This means that the value returns only after the thread that's retrieving the value is complete.
+The [hotspot=await file=1]`await` method stops and waits until all of the requests have completed. 
+While waiting for this countdown to complete, the main thread is free to perform other tasks. 
+In this case, no such task is present, but a simple task can be given to occupy the thread.
```

Comment:
I know I copied this directly from the original version but I'm not sure now what it means: "a simple task can be given..." - does it mean that the user should do something? We could remove from "but a simple task..." to the end of the sentence. The problem is that it's phrased very passively - it's not clear who, if anyone, should give the thread a simple task to do.